portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,41 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jennifer's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+    <header>
+      <h1>JENNIFER JIMENEZ</h1>
+    </header>
+    <nav>
+      <ul class="navbar">
+        <li>Home</li>
+        <li>Projects & Work</li>
+        <li>Gallery</li>
+      </ul>
+    </nav>
+    <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>
+        Hello! My name is Jennifer and I am rising junior at Yale University studying computer science.
+        I am a proud member of Trumbull College at Yale. On campus, I am involved with the Yale Undergraduate Prison 
+        Project as GED tutor for justice-impacted individuals.
+      <br></br>
```

Comment:
nit: I would recommend using 3 separate paragraph elements here, rather than one with `<br>` tags to separate content visually. It's a more accurate representation of the document's structure (these truly seem like 3 separate paragraphs with distinct themes), and there are [accessibility concerns with using `<br>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns) in this manner.

It also blurs our [separation of concerns](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns) if we're using the `<br>` tag in HTML just to control how the content is presented visually. Better to use CSS to modify the layout if the default styling applied to your paragraphs doesn't give you quite what you want.